PersonStageThree/src/com/beyond/person/core/exception/InvalidAgeException.java

```diff
@@ -0,0 +1,7 @@
+package com.beyond.person.core.exception;
+
+public class InvalidAgeException extends Exception {
```

Comment:
@maria-salmasian jan exceptions should not be age or name specific , you can have ValidationException and use in all validation exceptions this one with the corresponding message

Reply:
changed them all